src/App.vue

```diff
@@ -96,7 +96,6 @@ export default class App extends Mixins(DateMixin) {
 
   private filingData: Array<FilingDataIF> = []
   private totalFee: number = 0
-  private entityType: string = 'CP'
 
   @Action('setEntityType') setEntityType!: ActionBindingIF
```

Comment:
I know you didn't work on this in this PR but are we using this Binding in this component?

Reply:
Good catch! Will remove it. Thanks